src/options/generate/static_files/base_image/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM jinaai/jina:3.14.1-py39-standard
```

Comment:
we are using a different jina version now

Reply:
`FROM jinaai/jina:3.15.1-dev14-py39-standard`

Reply:
done thanks